test/test_torch.py

```diff
@@ -2620,6 +2620,26 @@ def test_index_add(self):
             dest2[idx[i]] = dest2[idx[i]] + src[i]
         self.assertEqual(dest, dest2)
 
+    # add coverage for issue with atomic add that appeared only for 
```

Comment:
It would be nice (doesn't have to happen in this PR) to change `test_index_add` to work with dtypes as well to add more test coverage.